pkg/sql/schemachanger/scop/immediate_mutation.go

```diff
@@ -429,6 +429,13 @@ type AddColumnFamily struct {
 	Name     string
 }
 
+// AssertColumnFamilyIsRemoved asserts that a column family is removed.
```

Comment:
Yeah, let me add the justification. My fear here is the disconnect between the element model since we want the transition to be official once it's actually removed.